lib/wickedbase.pm

```diff
@@ -223,7 +223,7 @@ sub ping_with_timeout {
     my ($self, %args) = @_;
     $args{ip_version}         //= 'v4';
     $args{timeout}            //= '60';
-    $args{proceed_on_failure} //= 0;
```

Comment:
To me it looks like we disabling that function, or?

Reply:
agree , I miss that . PR updated WDYT now ?

Reply:
I only found one user of this method, which use `proceed_on_failure =>1`

```
tests/wicked/basic/sut/t01_basic.pm:60:    die('IP should not be reachable') if ($self->ping_with_timeout(ip => '10.0.2.2', timeout => '2', proceed_on_failure => 1));
```
It looks like, we test there, that the connection to a specific host should _not_ work. I would keep the logic to check for failed connections.